docs/framework-luna_security_provider.md

```diff
@@ -92,6 +92,7 @@ The framework can be configured by modifying the [`config/luna_security_provider
 | Name | Description
 | ---- | -----------
 | `logging_enabled` | Whether to enable the logging wrapper for the Luna Security Provider.  Defaults to `false`.
+| `ha_logging_enabled` | Whether to enable HA logging for the Luna Security Provider.  Defaults to `true`.
```

Comment:
In general, we'd alphabetize these (for my OCD), but I'll do it in a following commit.